cves/CVE-2013-1665.yml

```diff
@@ -287,7 +344,7 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 200
```

Comment:
Upvote +1

Reply:
Overall very good CVE research, very detailed and well articulated. Although some grammar mistakes are present and have been previously pointed out.

Reply:
Thank you! I will correct those mistakes.